52/jread/test_logtimes.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime, timedelta
+
+from logtimes import loglines, convert_to_datetime, time_between_shutdowns
+
+
+def test_convert_to_datetime():
+    line1 = 'ERROR 2014-07-03T23:24:31 supybot Invalid user dictionary file'
+    line2 = 'INFO 2015-10-03T10:12:51 supybot Shutdown initiated.'
+    line3 = 'INFO 2016-09-03T02:11:22 supybot Shutdown complete.'
+    assert convert_to_datetime(line1) == datetime(2014, 7, 3, 23, 24, 31)
```

Comment:
You can use `pytest.mark.parametrize` - https://pybit.es/articles/pytest-coding-100-tests/ - no 9